### ISSUE TITLE
target_label is always zero

### ISSUE THREAD

**Issue by JiwenZ** [luyug/Reranker#10]
> self.register_buffer(
>       'target_label',
>       torch.zeros(self.train_args.per_device_train_batch_size, dtype=torch.long)
> )

I don't think target_label is given any value after initialization. So the labels are always zeros.
Please correct me if I'm wrong.

**Comment by JiwenZ** [comment]
I got it, that is how contrastive loss works

**Comment by simonhessner** [comment]
@JiwenZ @luyug I was wondering about this as well but I couldn't find any literature that explains why all labels are set to 0. Do you remember the reason for setting the targets to 0 no matter if the samples are positives or negatives?
